recipes/php.rb

```diff
@@ -41,7 +41,7 @@
 end
 
 
-template "/etc/php.d/zz-magento.ini" do
+template "/etc/php.d/zz-#{node['app']['name']}.ini" do
   source "zz-magento.ini.erb"
```

Comment:
No, because then we'd need a separate template file in git for each individual app name, which isn't the intention here.  We could make the template name `zz-appname.ini` or something similarly generic, but I don't really want to spend the time doing that right now.